package.json

```diff
@@ -99,5 +101,10 @@
   "peerDependencies": {
     "react": "^16.13.1",
     "react-dom": "^16.13.1"
+  },
+  "compileFonts": {
```

Comment:
could we separate the config (from package.json) into a different file? For example use https://github.com/davidtheclark/cosmiconfig ? This would also probably make parsing the config much clearer in the gulpfile ;)

Reply:
I think that is a good idea.  I will look into it.

Reply:
I have pushed another commit which moves the options to a separate file. (which is also added to the git ingnore) I have also improved the error messages for the options and updated the documentation.